x-pack/plugins/cases/public/components/connectors/thehive/case_fields.test.tsx

```diff
@@ -45,7 +44,7 @@ describe.skip('TheHive Cases Fields', () => {
       </MockFormWrapperComponent>
     );
 
-    userEvent.selectOptions(screen.getByTestId('tlp-field'), '4');
+    await userEvent.selectOptions(screen.getByTestId('tlp-field'), '4');
```

Comment:
Works for me!

Reply:
Accepted the change, but can you explain what difference does it make?

Reply:
`find` uses `waitFor` under the hood so this might help when `render` for whatever reason takes longer and not all options are immediately available.

I'm not sure this fixes anything in this case but it's a sanity check we like to add when something gets flaky on `cases` 😛